### PR TITLE
chore(javascripts): prefer-arrow-callback

### PR DESCRIPTION
**Pull request by firfircelik**
according to this issue https://github.com/up-for-grabs/up-for-grabs.net/issues/1515.

npm fix everything, sorry for that :)

### CODE REVIEW

**Comment by shiftkey** [comment]
> npm fix everything, sorry for that :)

Can you please follow the instructions in the original issue? There are a lot of changes introduced that are separate to the problem outlined, and there are CI issues introduced too...

File: javascripts/projectsService.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /* eslint global-require: "off" */
 /* eslint block-scoped-var: "off" */
+/* eslint prefer-arrow-callback: [ "error" ] */
```
Comment (shiftkey) [inline javascripts/projectsService.js]: Like we've found in #1520, I think we also need to add this, run `npm run prettier-fix` and then commit to make CI pass:


```suggestion
/* eslint prefer-arrow-callback: [ "error" ] */
/* eslint arrow-parens: [ "error", "as-needed" ] */
```

Comment (firfircelik) [reply javascripts/projectsService.js]: this is the running one. don't get why its failed.

**Comment by shiftkey** [comment]
I think the errors you're now seeing (which I didn't expect) are due to clashes between eslint and prettier.

Add these two rules to the top of the file, and I'll write up an issue to track how we can avoid these clashes in the future:

```js
/* eslint function-paren-newline: [ "off" ] */
/* eslint implicit-arrow-linebreak: [ "off" ] */
```

**Comment by shiftkey** [comment]
@firfircelik thanks for working through these issues with me.

I'm gonna pause and revisit how we use eslint and prettier to see if we can reduce the pain here with upgrading to prevent these "feedback loop" events from happening again.

**Pull request merged**